spek-runtime/jvm/src/main/kotlin/org/spekframework/spek2/runtime/SpekJvmRuntime.kt

```diff
@@ -58,4 +52,19 @@ actual class SpekRuntime: AbstractRuntime() {
             .map { it.objectInstance ?: it.primaryConstructor!!.call() }
             .firstOrNull() ?: defaultInstanceFactory
     }
+
+    private fun createReflections(testDirs: List<String>): Reflections {
+        val urls = if (testDirs.isEmpty()) {
+            ClasspathHelper.forClassLoader()
+        } else {
+            testDirs.map { java.nio.file.Paths.get(it) }
```

Comment:
Isn’t it better just pass `File` instances instead of `String`?

Reply:
👍